data/CMakeLists.txt

```diff
@@ -30,9 +30,9 @@ if (RELEASE)
         ${CMAKE_SOURCE_DIR}/README.md
         ${CMAKE_SOURCE_DIR}/LICENSE.txt
         ${CMAKE_SOURCE_DIR}/LICENSE-OFL-Terminus-Font.txt
-        DESTINATION ${CMAKE_INSTALL_PREFIX})
-    install(DIRECTORY ${CMAKE_SOURCE_DIR}/doc DESTINATION ${CMAKE_INSTALL_PREFIX})
+        DESTINATION ${README_DIR})
+    install(DIRECTORY ${CMAKE_SOURCE_DIR}/doc DESTINATION ${DATA_PREFIX})
     if (TILES)
-        install(DIRECTORY ${CMAKE_SOURCE_DIR}/gfx DESTINATION ${CMAKE_INSTALL_PREFIX})
```

Comment:
So, this is weird. The old code matches the layout of eg. the cataclysm mac bundle. Inside the bundle we have `cataclysm-tiles, data, gfx, lang`. At the very least, thing should end up in the release at the correct relative path compared to their source, so we shouldn't be changing any of these.

If we *are* going to change them, we should maybe move the non-data related install calls to the root CMakeLists.txt file which makes more sense than some random subfolder touching these.

Reply:
Fixed by removing [7ccc28e](https://github.com/CleverRaven/Cataclysm-DDA/pull/61880/commits/7ccc28ed5be410b16526e9708c97b6f239d401c9).